test/e2e/gpu/gpu.go

```diff
@@ -98,7 +98,7 @@ func describe() {
 		framework.ExpectNoError(err, "pod Create API error")
 
 		ginkgo.By("waiting the pod to finish successfully")
-		e2epod.NewPodClient(f).WaitForSuccess(ctx, pod.ObjectMeta.Name, 60*time.Second)
+		errPodNotSuccess := e2epod.WaitForPodSuccessInNamespace(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name)
 
 		ginkgo.By("checking log output")
```

Comment:
While at it, you could add gpu log output as well.

Reply:
fixed!